packages/fxa-content-server/app/styles/modules/_cad.scss

```diff
@@ -0,0 +1,55 @@
+#fxa-connected {
```

Comment:
Just a heads up that any SCSS written here I'll probably be refactoring to Tailwind classes with [FXA-5454](https://mozilla-hub.atlassian.net/browse/FXA-5454). Don't spend a ton of time refactoring, but any existing classes you can use here (like `flex`) on elements will help me out. 😁 (you may need to rebase against `main`)